src/server/core/Server.js

```diff
@@ -77,6 +87,9 @@ Server.prototype.onSocketDisconnection = function(client)
 {
     console.info('Client %s disconnected.', client.id);
 
+    // Notify clients with updated nb players
+    this.curvytronController.sendNbPlayers();
```

Comment:
This is not for the Server to handle that kind of stuff: the controller in charge of "nbPlayers" should listen for client connection/disconnection and send the events itself.